pgmpy/base/DAG.py

```diff
@@ -71,7 +72,10 @@ class DAG(nx.DiGraph):
     3
     """
 
-    def __init__(self, ebunch=None, latents=set()):
+    def __init__(self, ebunch=None, latents=set(), lavaan_str=None, syntax=None):
+        if syntax == "lavaan":
```

Comment:
I would suggest removing the `syntax` argument and instead check if `lavaan_str` is specified or not.